cmd/crc/cmd/version.go

```diff
@@ -37,15 +37,13 @@ type version struct {
 	Version          string `json:"version"`
 	Commit           string `json:"commit"`
 	OpenshiftVersion string `json:"openshiftVersion"`
-	PodmanVersion    string `json:"podmanVersion"`
```

Comment:
this should not be removed, the podman version here is also used in the  `openshift` and `microshift` presets

Reply:
Thanks! Added it back in second commit :)